apps/fluent-tester/src/TestComponents/Icon/IconExperimental.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FontIcon, SvgIcon, Icon } from '@fluentui-react-native/experimental-icon';
+import { Text, View } from 'react-native';
+import TestSvg from './assets/test.svg';
+import { SvgIconProps } from '@fluentui-react-native/experimental-icon';
+
+const fontBuiltInProps = {
+  fontFamily: 'Arial',
+  codepoint: 0x2663,
+  fontSize: 100,
+  color: '#f09',
+};
+
+const svgUriProps: SvgIconProps = {
+  uri: 'https://upload.wikimedia.org/wikipedia/commons/8/84/Example.svg',
+  viewBox: '0 0 1000 1000',
+  width: 100,
+  height: 100,
+};
+
+const svgSrcProps: SvgIconProps = {
+  viewBox: '0 0 500 500',
+  src: TestSvg,
+  width: 72,
+  height: 72,
+};
+
+export const IconExperimental: React.FunctionComponent = () => {
+  return (
+    <View>
+      <Text>Icon component</Text>
+      <Icon svgSource={{ color: 'lightgreen', ...svgSrcProps }} />
```

Comment:
Please useMemo for the object here and the 3rd icon to avoid recreation on every render.